src/debugger/server-logic/Constants.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+/**
+ * Configuration keys for Server Logic settings
+ */
+export const SERVER_LOGIC_CONFIG_KEYS = {
+    DONT_SHOW_WELCOME: 'powerPages.serverLogic.dontShowWelcome'
+} as const;
+
+/**
+ * File and folder names used by Server Logic debugger
+ */
+export const SERVER_LOGIC_FILES = {
+    RUNTIME_LOADER: 'server-logic-runtime-loader.js',
+    SERVER_LOGICS_FOLDER: 'server-logics',
+    VSCODE_FOLDER: '.vscode',
+    GITIGNORE: '.gitignore'
+} as const;
+
+/**
+ * URLs for Server Logic documentation
+ */
+export const SERVER_LOGIC_URLS = {
+    LEARN_MORE: 'https://go.microsoft.com/fwlink/?linkid=2346212'
+} as const;
+
+/**
+ * Regex pattern for matching server-logics folder in file paths
+ */
+export const SERVER_LOGICS_FOLDER_PATTERN = /[/\\]server-logics[/\\]/;
```

Comment:
I believe this will change to just `server-logic`. Better to check for both to avoid changing it in future,